packages/perseus/src/interactive2/arrowhead.ts

```diff
@@ -0,0 +1,96 @@
+import {vector as kvector} from "@khanacademy/kmath";
+import _ from "underscore";
+
+import KhanMath from "../util/math";
+
+import {getClipPoint} from "./get-clip-point";
+import WrappedPath from "./wrapped-path";
+
+import type {Graphie} from "../util/graphie";
+import type {Coord} from "@khanacademy/perseus";
+
+export class Arrowhead extends WrappedPath {
+    private static scale = 1.4;
+    center: Coord;
+
+    constructor(graphie: Graphie, style: any) {
+        // Points that define the arrowhead
+        const center: Coord = [0.75, 0];
+        let points: Coord[] = [
+            [-3, 4],
+            [-2.75, 2.5],
+            [0, 0.25],
+            center,
+            [0, -0.25],
+            [-2.75, -2.5],
+            [-3, -4],
+        ];
+
+        // Scale points by `Arrowhead.scale` around `center`
+        points = _.map(points, function (point) {
+            const pv = kvector.subtract(point, center);
+            const pvScaled = kvector.scale(pv, Arrowhead.scale);
+            return kvector.add(center, pvScaled);
+        });
+
+        // We can't just pass in a path to `graph.fixedPath` as we need to modify
+        // the points in some way, so instead we provide a function for creating
+        // the path once the points have been transformed
+        const createCubicPath = function (points) {
+            let path = "M" + points[0][0] + " " + points[0][1];
+            for (let i = 1; i < points.length; i += 3) {
+                path +=
+                    "C" +
+                    points[i][0] +
+                    " " +
+                    points[i][1] +
+                    " " +
+                    points[i + 1][0] +
+                    " " +
+                    points[i + 1][1] +
+                    " " +
+                    points[i + 2][0] +
+                    " " +
+                    points[i + 2][1];
+            }
+            return path;
+        };
```

Comment:
TODO: move this function outside the class so the scopes are clearer. `points` shadows the variable from the outer scope.